modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -30,6 +30,20 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
         });
       };
 
+      const fireEvent = (elem: SugarElement<Node>, event: string) => {
```

Comment:
This isn't something to act on in this PR, but we really need to move this to Agar at some stage soon.

Reply:
Logged a ticket for this